src/orderFeeds/ws.order.feed.ts

```diff
@@ -118,7 +130,8 @@ export class WsNextOrder extends GetNextOrder {
 
     this.socket.on("close", async () => {
       this.logger.error(`WsConnection is closed`);
-      await setTimeout(5000);
+      clearTimeout(this.pingTimer);
+      await setTimeoutPromise(5000);
```

Comment:
Honestly, I don't like the idea of re-connection on `close` event: this means you cannot mindfully close the connection. Let's move the re-connection trigger heartbeat handler, right after the current connection is terminated.